dsl_parser/utils.py

```diff
@@ -21,15 +21,9 @@
 import sys
 import re
 
-try:
```

Comment:
this is unrelated, but since i already made a compat module, lets just move this now